Mark some slowish MMS tests as heavy

lindsayad opened the pull request:
Help speed up testing

lindsayad committed:
Mark some slowish MMS tests as heavy

refs #000

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/850766/) on 326322f wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/19117/site)

This comment will be updated on new commits.

moosebuild commented:
Job [Generate and verify coverage](https://civet.inl.gov/job/850778/) on 326322f wanted to post the following:


## Framework coverage

<table>
  <thead>
    <tr>
      <th></th>
      <th>720603</th>
      <th colspan=4>#19117 326322</th>
    </tr>
    <tr>
      <th></th>
      <th>Total</th>
      <th>Total</th>
      <th>+/-</th>
      <th>New</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>Rate</b></td>
      <td>81.72%</td>
      <td>81.71%</td>
      <td>-0.01%</td>
      <td>-</td>
    </tr>
    <tr>
      <td><b>Hits</b></td>
      <td>70503</td>
      <td>70494</td>
      <td>-9</td>
      <td>0</td>
    </tr>
    <tr>
      <td><b>Misses</b></td>
      <td>15775</td>
      <td>15784</td>
      <td>+9</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

[Diff coverage report](https://mooseframework.inl.gov/docs/PRs/19117/coverage_diff/framework.html)

[Full coverage report](https://mooseframework.inl.gov/docs/PRs/19117/coverage/framework)


## Modules coverage

Coverage did not change

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/contact)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/geochemistry)
- [`heat_conduction`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/heat_conduction)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/navier_stokes)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/richards)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/stochastic_tools)
- [`tensor_mechanics`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/tensor_mechanics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/19117/coverage/xfem)

</details>



This comment will be updated on new commits.

lindsayad committed:
Add coarse adapt test

This ensures that we have coverage of failed system inversions during FV
gradient computation in non-heavy testing
